tests/test_actions.py

```diff
@@ -22,22 +22,33 @@
 from screenpy.resolutions import IsEqualTo
 from conftest import mock_settings
 from unittest_protocols import ErrorQuestion
-from useful_mocks import get_mock_action, get_mock_question, get_mock_resolution
+from useful_mocks import (
+    get_mock_action,
+    get_mock_question,
+    get_mock_resolution,
+    FakeQuestion,
+)
+
+
+ACTION = get_mock_action()
+# QUESTION = get_mock_question()
+QUESTION = FakeQuestion()
+RESOLUTION = get_mock_resolution()
```

Comment:
I have some concerns about using the mocks like this, since the mock object will accumulate calls throughout all the tests in this file. Maybe it makes more sense to make a `pytest.fixture` for these?

Reply:
Take a look at the latest commits.  There are no longer any reused mocks.